### DISCUSSION TITLE
Create a gif from images in a directory.

### DISCUSSION THREAD

**Discussion by taextreme** [bytedeco/javacv#1820] [Q&A] [answered]
I am currently implement console app that take in a video and make a gif from that video. 

Right now, I was able to extract several frames from a video and save it as images using  FFmpegFrameGrabber and ImageIO.

So, I want to ask is there a way to make those images into a GIF?

**Comment by saudet** [comment, marked as answer]
We can use FFmpegFrameRecorder for that, but it's probably easier to just use the ffmpeg program:
http://bytedeco.org/javacpp-presets/ffmpeg/apidocs/org/bytedeco/ffmpeg/ffmpeg.html
